array-manipulation-2/src/drop.js

```diff
@@ -0,0 +1,15 @@
+/* exported drop */
+
+// Enters loop, assign var i to count, set condition i < array.length
+// Push elements to new array
+
+function drop(array, count) {
+  var newArray = [];
+  for (var i = count; i < array.length; i++) {
+    if (count < array.length) {
```

Comment:
Same thing here, you should be able to remove this conditional and keep the work inside it.